File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Enter a positive integer: ");
    }
    while (h < 1 || h > 8);
    for (int i = 0; i < h; i++)
    {
        for (int j = 1; j < h - i; j++)
        {
            printf(" ");
        }
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario.py
from cs50 import get_int

while True:
    h = get_int("Height: ")
    
    if h > 0 and h < 9:
        break
    
for i in range(1, h + 1):
    for j in range(h - i):
        print(" ", end="")
    for k in range(i):
        print("#", end="")
print("")
